PRS7001-8000/PRS7597Nob.xml

```diff
@@ -32,26 +32,38 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="en">Gǝʿǝz</language>
+         </langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="MKr" when="2024-09-20">Added floruit</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Nob
-               </persName>
-               <occupation> abbot of Rema </occupation>
+               <persName xml:lang="gez" xml:id="n1">ኖብ፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">Nob</persName>
+               <floruit>He was the brother of ʾaṣe <persName ref="PRS10342Yeshaq">Yǝsḥāq</persName> and spiritual son of <persName ref="PRS10253Yasay">Yāsāy</persName>.
+             He is credited as the founder of the monastic community of <ref type="ins" corresp="INS0359Rema">Remā</ref>.</floruit>
             </person>
+            <listRelation>
+               <relation name="snap:BrotherOf" active="PRS7597Nob" passive="PRS10342Yeshaq"/>
+           </listRelation>
```

Comment:
you can also add already here the relation formerlyListedAs to PRS14216NobWaldaYasey (and possibly also delete PRS14216NobWaldaYasey as suggested in https://github.com/BetaMasaheft/Documentation/issues/2650) to save the number of branches/pull requests

Reply:
Possibly the whole string Nob Walda Yasay can be added also as n2